cmd/krel/cmd/ff.go

```diff
@@ -93,7 +102,7 @@ func runFf(opts *ffOptions) error {
 
 	cleanup := rootOpts.cleanup
 	if cleanup {
-		defer repo.Cleanup() // nolint: errcheck
+		defer func() { err = repo.Cleanup() }()
```

Comment:
This stomps out an error if repo.Cleanup() succeeds.

Reply:
Right, I will revert my change :+1: